test/end-to-end/company.js

```diff
@@ -450,6 +466,51 @@ describe("Company endpoint", () => {
                 expect(res.body).toHaveProperty("offers");
                 expect(res.body.offers.length).toEqual(0);
             });
+
+            test("should return if a disabled company returns 403", async () => {
```

Comment:
Change the test description to "Should fail to retrieve details if company is disabled"